west.yml

```diff
@@ -89,7 +89,7 @@ manifest:
       revision: bc62a2fa9d98ddb5d633c932ea199bc68e10f194
       path: modules/fs/nffs
     - name: hal_nxp
-      revision: 81d50b7d15340dfa034bfff9c98527f5b661b84e
+      revision: pull/3/head
```

Comment:
Need to update this now that zephyrproject-rtos/hal_nxp#3 has been merged

Reply:
Done